apps/web/src/components/common/EnhancedTable/index.tsx

```diff
@@ -187,7 +187,7 @@ function EnhancedTable({ rows, headCells, mobileVariant }: EnhancedTableProps) {
         </Table>
       </TableContainer>
 
-      {rows.length > pagedRows.length && (
+      {(rows.length > pageSizes[0] || rowsPerPage !== pageSizes[1]) && (
```

Comment:
Whilst this works, I think we could avoid relying on default values or array order with something like:

```suggestion
      {rows.length > rowsPerPage && (
```

Reply:
@iamacook your suggested code doesn't work, the pager disappears.